code/go/0chain.net/smartcontract/minersc/service.go

```diff
@@ -0,0 +1,38 @@
+package minersc
+
+import (
+	"strconv"
+
+	cstate "0chain.net/chaincore/chain/state"
+	"0chain.net/chaincore/transaction"
+	"0chain.net/core/common"
+	"0chain.net/core/config"
+	common2 "0chain.net/smartcontract/common"
+)
+
+// killMiner
+// killing is permanent and a killed miner cannot receive any rewards
+func (_ *MinerSmartContract) addHardFork(
+	txn *transaction.Transaction,
+	input []byte,
+	gn *GlobalNode,
+	balances cstate.StateContextI,
+) (resp string, err error) {
+	var changes config.StringMap
+	if err = changes.Decode(input); err != nil {
+		return "", common.NewError("add_hardfork", err.Error())
+	}
+
+	for key, val := range changes.Fields {
```

Comment:
Map iterating is random, let's sort before using it.

Reply:
what is the reason for it? not sure it can break anything